script.js

```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
+  './images/Proj3-bg.png',
+  [['html'], ['bootstrap'], ['Ruby']], '#', '#', 'project project4');
+
+const featuredWork = document.createElement('div');
+featuredWork.className = featuredProject.className;
+recentWorks.appendChild(featuredWork);
+
+const gymfit = document.createElement('div');
+gymfit.className = 'gymfit';
+featuredWork.appendChild(gymfit);
+
+const gymfitImg = document.createElement('img');
+gymfitImg.src = featuredProject.featuredImage;
+gymfitImg.style.width = '100%';
+gymfit.appendChild(gymfitImg);
+
+const multiPostStories = document.createElement('div');
+multiPostStories.className = 'desk-feat-description';
+featuredWork.appendChild(multiPostStories);
+
+const mpsTitle = document.createElement('div');
+mpsTitle.className = 'post-stories';
+mpsTitle.innerHTML = featuredProject.name;
+multiPostStories.appendChild(mpsTitle);
+
+const featText = document.createElement('div');
+featText.className = 'feat-text';
+featText.innerHTML = featuredProject.description;
+multiPostStories.appendChild(featText);
+
+const featLangList = document.createElement('ul');
+featLangList.className = 'lang-list-4';
+multiPostStories.appendChild(featLangList);
+
+const [tech1, tech2, tech3, tech4] = featuredProject.technologies;
+
+const featHtml = document.createElement('li');
+featHtml.className = 'html';
+featHtml.innerHTML = tech1;
+featLangList.appendChild(featHtml);
+
+const featCss = document.createElement('li');
+featCss.className = 'css';
+featCss.innerHTML = tech2;
+featLangList.appendChild(featCss);
+
+const featbootstrap = document.createElement('li');
+featbootstrap.className = 'bootstrap';
+featbootstrap.innerHTML = tech3;
+featLangList.appendChild(featbootstrap);
+
+const featRuby = document.createElement('li');
+featRuby.className = 'Ruby';
+featRuby.innerHTML = tech4;
+featLangList.appendChild(featRuby);
+
+const featSeeProject = document.createElement('button');
+featSeeProject.className = 'see-project';
+featSeeProject.innerHTML = '<h2>See Project<h2>';
+featSeeProject.style.width = '30%';
+multiPostStories.appendChild(featSeeProject);
+
+const projectsSection = document.createElement('div');
+projectsSection.className = 'projects';
+recentWorks.appendChild(projectsSection);
+
+featSeeProject.addEventListener('click', () => {
+  const projectPopup = document.createElement('div');
+  projectPopup.className = 'project-popup';
+  body.appendChild(projectPopup);
+
+  const overlay = document.createElement('div');
+  overlay.id = 'overlay';
+  body.appendChild(overlay);
+
+  const popupTitle = document.createElement('div');
+  popupTitle.className = 'popup-title';
+  popupTitle.innerHTML = featuredProject.name;
+  projectPopup.appendChild(popupTitle);
+
+  const popupClose = document.createElement('img');
+  popupClose.className = 'popup-close';
+  popupClose.src = './images/close.png';
+  popupTitle.appendChild(popupClose);
+
+  const popupLangs = document.createElement('ul');
+  popupLangs.className = 'popup-buttons';
```

Comment:
- [ ] Please implement styling for buttons, hover and pressed 
![image](https://user-images.githubusercontent.com/62705947/217575487-b4be14de-7d16-4358-9261-9fdcac9b4a15.png)